gramps/plugins/lib/libmixin.py

```diff
@@ -77,8 +83,8 @@ def __find_primary_from_handle(
         obj = class_type()
         handle = str(handle)
         new = True
-        raw = get_raw_obj_data(handle)
-        if raw is not None:
+        if has_obj_handle(handle):
+            raw = get_raw_obj_data(handle)
             obj = data_to_object(raw)
```

Comment:
I can't see why this doesn't just get the object in one go.

Reply:
Addressed in the commit
>__find_primary_from_handle no longer uses get_raw_obj_from_handle to check for handle existence. As a result it can get the object directly rather then convert the raw data into an object